### PR TITLE
Add copyright notices to source files missing them

### PR DESCRIPTION
**Pull request by PetteriAimonen**
I have been asked by Rabbit Hole Computing to submit this pull request to clarify the copyright on files that I have written under contract to them. It didn't occur to me at the time to add the notices, even though it is generally recommended.

This pull request adds the notices to the source files that are my work. I consider that most of the headers don't have much copyrightable content in them. If you prefer, I can add the notices to them also for consistency.

Also if you'd prefer the full GPL license declaration at the beginning of the files, I can modify this pull request.

### CODE REVIEW

File: lib/BlueSCSI_platform_RP2040/scsiPhy.cpp [hunk 1]
```diff
@@ -1,5 +1,8 @@
 // Implements the low level interface to SCSI bus
 // Partially derived from scsiPhy.c from SCSI2SD-V6
+//
+// Copyright (c) 2022 Rabbit Hole Computing™
+// Copyright (c) 2014 Michael McMaster
```
Comment (erichelgeson) [inline lib/BlueSCSI_platform_RP2040/scsiPhy.cpp:5]: This notice does not match your notice. https://github.com/ZuluSCSI/ZuluSCSI-firmware/blob/main/lib/ZuluSCSI_platform_RP2040/scsiHostPhy.cpp

Comment (PetteriAimonen) [reply lib/BlueSCSI_platform_RP2040/scsiPhy.cpp:5]: Hmm, one of those is "scsiPhy.cpp" which is derived from SCSI2SD, as noted also in [ZuluSCSI version of the file](https://github.com/ZuluSCSI/ZuluSCSI-firmware/blob/main/lib/ZuluSCSI_platform_RP2040/scsiPhy.cpp). The one in your link is "scsiHostPhy.cpp" for the host/initiator mode that is independent implementation. I think the copyright comments here are correct?

Comment (erichelgeson) [reply lib/BlueSCSI_platform_RP2040/scsiPhy.cpp:5]: You are correct, I was looking at host vs non.

**Comment by PetteriAimonen** [comment]
Any updates on this?

**Comment by PetteriAimonen** [comment]
Rabbit Hole Computing has asked me to inquire whether there is anything that could be done to get this merged without involving lawyers.

**Comment by erichelgeson** [comment]
> Rabbit Hole Computing has asked me to inquire whether there is anything that could be done to get this merged **without involving lawyers**.

RHC's lawyer emailed me on Fri, Apr 7, 2:13 PM CST regarding this PR that was opened April 5th, 1:21AM CST. 

I've responded to the email from their lawyer and received no response back regarding this PR.

So to their question there is nothing to be done _without involving lawyers_ as they have done so already just days after the PR was open. I will be closing this PR and locking it as there is no need to continue this via this forum, as RHC chose a different path a month ago.

**Pull request closed without merging**